client/js/components/EnvironmentVariables/EnvironmentVariables.less

```diff
@@ -0,0 +1,22 @@
+// EnvironmentVariables.less
+@import "../../../less/variables";
+
+.environmentVariables {
+	overflow-y: scroll !important;
+}
+
+.environmentVariables-Notice--error {
+	background-color: @color-error-bg !important;
```

Comment:
@ifandelse We've been following our convention for the most part of naming variables according to the module, and mapping them in the variables.less file. So `@color-enviromentVariables-error-bg` etc, though I wonder if its a little overboard since we probably won't be theming this project…